packages/react-reconciler/src/ReactFiberUnwindWork.js

```diff
@@ -80,8 +81,12 @@ function unwindWork(
     }
     case DehydratedSuspenseComponent: {
       if (enableSuspenseServerRenderer) {
-        // TODO: popHydrationState
         popSuspenseContext(workInProgress);
+        if (workInProgress.alternate === null) {
+          // TODO: popHydrationState
```

Comment:
I can't actually do this yet because this because popping causes hydration warnings. This is part of a larger bug that suspending in a dehydrated tree can leave the hydration context in an inconsistent state when we move to work on the sibling of the thing that suspended. That's a bigger issue to fix.